### PR TITLE
static pages

### PR DESCRIPTION
**Pull request by namson168**
I have created new pull request, check it please

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -18,6 +18,7 @@ rerun.txt
 pickle-email-*.html
 .project
 /config/database.yml
+/config/database.yml.example
```
Comment (dieunb) [inline .gitignore]: remove this line

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -1,2 +1,11 @@
 module ApplicationHelper
+  # Returns the full title on a per-page basis.
+  def full_title page_title = ""
+    base_title = t("base_title")
+    if page_title.empty?
+      base_title
+    else
+      page_title + " | " + base_title
+    end
```
Comment (lehongson12) [inline app/helpers/application_helper.rb]: page_title.empty? ? base_title : "#{page_title} | #{base_t

File: app/views/static_pages/about.html.erb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<% provide :title, t("about") %>
+<h1><%= t("about") %></h1>
+<p>
+  The <a><em><%= link_to "Ruby on Rails Tutorial",
```
Comment (lehongson12) [inline app/views/static_pages/about.html.erb]: I18n

File: app/views/static_pages/about.html.erb [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+  "http://www.railstutorial.org/"%></em></a> is a
+  <a><%= link_to "book", "http://www.railstutorial.org/book"%></a> and
+  <a><%= link_to "screencast series", "http://screencasts.railstutorial.org/"%></a>
```
Comment (lehongson12) [inline app/views/static_pages/about.html.erb]: i18n

File: app/views/static_pages/about.html.erb [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+  to teach web development with
+  <a><%= link_to "Ruby on Rails", "http://rubyonrails.org/"%></a>.
+  This is the sample application for the tutorial.
```
Comment (lehongson12) [inline app/views/static_pages/about.html.erb]: i18n

File: app/views/static_pages/contact.html.erb [hunk 6]
```diff
@@ -0,0 +1,6 @@
+<% provide :title, t("contact") %>
+<h1><%= t("contact") %></h1>
+<p>
+  Contact the Ruby on Rails Tutorial about the sample app at the
```
Comment (lehongson12) [inline app/views/static_pages/contact.html.erb]: i18n

File: app/views/static_pages/contact.html.erb [hunk 7]
```diff
@@ -0,0 +1,6 @@
...
+  <a><%= link_to "contact page", "http://www.railstutorial.org/#contact"%></a>.
```
Comment (lehongson12) [inline app/views/static_pages/contact.html.erb]: i18n

File: app/views/static_pages/help.html.erb [hunk 8]
```diff
@@ -0,0 +1,10 @@
+<% provide :title, t("help") %>
+<h1><%= t("help") %></h1>
+<p>
+  Get help on the Ruby on Rails Tutorial at the
+  <a><%= link_to "Rails Tutorial help section", 
+  "http://www.railstutorial.org/#help"%></a>.
+  To get help on this sample app, see the
+  <a><em><%= link_to "Ruby on Rails Tutorial", "http://www.railstutorial.org/book"%></em>
+  book</a>.
+</p>
```
Comment (lehongson12) [inline app/views/static_pages/help.html.erb]: i18n line 4 -> 8

**Pull request merged**